sphinx/builders/latex/__init__.py

```diff
@@ -463,6 +463,8 @@ def install_packages_for_ja(app: Sphinx) -> None:
 
 def default_latex_engine(config: Config) -> str:
```

Comment:
If `config` can be `None` then this type signature is wrong and should be changed to `Optional[Config]`

Reply:
@danieleades - `config.language` can be None, `config` it self is not `None`. Signature seems correct.

Reply:
Ah, my bad